lib/assessment.js

```diff
@@ -32,14 +42,14 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
-
+  return array.filter(callback);
```

Comment:
Using the built-in array method in your implementation of the same array method is cheating! It's good that you know about it and how to use it, though. No credit.